app/assets/stylesheets/ama_layout/layout_components/forms.scss

```diff
@@ -481,9 +481,25 @@ input[type="url"]:-ms-input-placeholder{
     @include large-8;
   }
 
+  &__input{
+    &--inline{
+      @include medium-8;
+    }
+  }
+
+  &__label{
+    &--white{
+      color:white;
+    }
+  }
+
   &__submit{
     &--right{
       @include large-4;
     }
+
+    &--inline{
```

Comment:
@sineade For the submit, I think we could, but for the input we need it to override the regular input class that makes it full width. Since we have to leave both, the simple form file is a bit more connected if they have similar class names. Does that sound okay to leave it?